examples/Cargo.lock

```diff
@@ -3321,6 +3321,7 @@ dependencies = [
  "serde",
  "serde-name",
  "serde_bytes",
+ "serde_json",
```

Comment:
That's probably not needed?

Reply:
That is needed for use of `serde_json::*` in code.